drs/startup.sh

```diff
@@ -1,5 +1,5 @@
-mkdir -p /drs/chord_drs/data/obj;
-mkdir -p /drs/chord_drs/data/db;
-
+cd chord_drs
 flask db upgrade;
-flask run --host=0.0.0.0;
\ No newline at end of file
+cd ..
+# flask run --host=0.0.0.0;
+gunicorn chord_drs.app:application -w 1 --threads $(expr 2 \* $(nproc --all) + 1) -b 0.0.0.0:${INTERNAL_PORT}
```

Comment:
Nitpicking. 
It should be possible to remain in the same directory (i.e. `chord_drs`) and invoke `gunicorn app:application -w 1 --threads $(expr 2 \* $(nproc --all) + 1) -b 0.0.0.0:${INTERNAL_PORT}` instead (remove the `chord_drs.` part)

Reply:
after setting `startup.sh` to
```
cd chord_drs
flask db upgrade;
pwd
ls -lah
# flask run --host=0.0.0.0;
gunicorn app:application -w 1 --threads $(expr 2 \* $(nproc --all) + 1) -b 0.0.0.0:${INTERNAL_PORT}
```

and despite clearly being in the right directory, for whatever reason, I get

```
/drs/bento_drs/chord_drs
total 76K    
drwxr-xr-x    4 root     root        4.0K Aug 10 19:27 .
drwxr-xr-x    1 root     root        4.0K Aug 10 19:40 ..
-rw-r--r--    1 root     root         232 Aug 10 19:27 __init__.py
-rw-r--r--    1 root     root        1.6K Aug 10 19:27 app.py
-rw-r--r--    1 root     root         621 Aug 10 19:27 backend.py
drwxr-xr-x    2 root     root        4.0K Aug 10 19:27 backends
-rw-r--r--    1 root     root        1.9K Aug 10 19:27 commands.py
-rw-r--r--    1 root     root        2.4K Aug 10 19:27 config.py
-rw-r--r--    1 root     root         235 Aug 10 19:27 constants.py
-rw-r--r--    1 root     root         455 Aug 10 19:27 data_sources.py
-rw-r--r--    1 root     root          77 Aug 10 19:27 db.py
-rw-r--r--    1 root     root         126 Aug 10 19:27 metrics.py
drwxr-xr-x    3 root     root        4.0K Aug 10 19:27 migrations
-rw-r--r--    1 root     root        3.4K Aug 10 19:27 models.py
-rw-r--r--    1 root     root         156 Aug 10 19:27 package.cfg
-rw-r--r--    1 root     root       11.4K Aug 10 19:27 routes.py
-rw-r--r--    1 root     root         283 Aug 10 19:27 utils.py
[2022-08-10 19:40:18 +0000] [22] [INFO] Starting gunicorn 20.1.0
[2022-08-10 19:40:18 +0000] [22] [INFO] Listening at: http://0.0.0.0:5000 (22)
[2022-08-10 19:40:18 +0000] [22] [INFO] Using worker: gthread
[2022-08-10 19:40:18 +0000] [24] [INFO] Booting worker with pid: 24
[2022-08-10 19:40:19 +0000] [24] [ERROR] Exception in worker process
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/site-packages/gunicorn/arbiter.py", line 589, in spawn_worker
    worker.init_process()
  File "/usr/local/lib/python3.6/site-packages/gunicorn/workers/gthread.py", line 92, in init_process
    super().init_process()
  File "/usr/local/lib/python3.6/site-packages/gunicorn/workers/base.py", line 134, in init_process
    self.load_wsgi()
  File "/usr/local/lib/python3.6/site-packages/gunicorn/workers/base.py", line 146, in load_wsgi
    self.wsgi = self.app.wsgi()
  File "/usr/local/lib/python3.6/site-packages/gunicorn/app/base.py", line 67, in wsgi
    self.callable = self.load()
  File "/usr/local/lib/python3.6/site-packages/gunicorn/app/wsgiapp.py", line 58, in load
    return self.load_wsgiapp()
  File "/usr/local/lib/python3.6/site-packages/gunicorn/app/wsgiapp.py", line 48, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/usr/local/lib/python3.6/site-packages/gunicorn/util.py", line 359, in import_app
    mod = importlib.import_module(module)
  File "/usr/local/lib/python3.6/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 994, in _gcd_import
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/drs/bento_drs/chord_drs/app.py", line 7, in <module>
    from chord_drs.backend import close_backend
ModuleNotFoundError: No module named 'chord_drs'
```

It's as if the project _has_ to be invoked from a directory above